src/main/java/com/dev/movietheatre/Main.java

```diff
@@ -58,10 +58,10 @@ public static void main(String[] args) throws AuthenticationException {
         User registeredUser1 = authenticationService.register("Sam@ukr.net", "000");
         shoppingCartService.addSession(movieSession,registeredUser1);
         ShoppingCart shoopingCartbyUser1 = shoppingCartService.getByUser(registeredUser1);
```

Comment:
please, still change the name of this variable

Reply:
renamed, done